src/RRTStar/obstacles.cpp

```diff
@@ -0,0 +1,74 @@
+#include "obstacles.h"
+
+#include <iostream>
+
+Obstacles::Obstacles()
+{
+}
+
+/**
+ * @brief Obstacles are stored as rectangles. Rectangle is denoted by two points : topLeft and bottomRight.
+ * @param firstPoint
+ * @param secondPoint
+ */
+void Obstacles::addObstacle(Vector2f firstPoint, Vector2f secondPoint)
```

Comment:
Can you please respect the Google function and variable naming guidelines? https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
Done! For some reason I thought C++ was camel case, thanks for the clarification.